### ISSUE TITLE
The inputs are float

### ISSUE THREAD

**Issue by moskomule** [itayhubara/BinaryNet.pytorch#2]
Hello, thank you for making PyTorch version of Binary Networks available. Now it gets easier to do research.

In the paper, the input features are given below.

![2018-01-15 11 40 05](https://user-images.githubusercontent.com/11806234/34924784-e0a8116c-f9e8-11e7-9977-b44a8f2a98e6.png)

But in this implementation, the inputs are float like [0.26962968707084656, 0.14762534201145172, -1.804444432258606...]. I just print the input features.

I'd like to know your idea on this. Thank you.

**Comment by itayhubara** [comment]
You are right! the inputs here are float. However the input tensor contains only 256 unique values (as we started from uint8 images). If you wish you can remove the normalization  float numbers were You can remove the normalization (alexnet_binary file line 65) and use integer values instead:
normalize = transforms.Normalize(mean=[128,128,128], std=[128,128,128])
I believe the results would be in the same ballpark

**Comment by moskomule** [comment]
Thank you for your reply. I'll try.
